content/resources.json

```diff
@@ -239,7 +239,76 @@
           "title": "🔷 Gaia-X Test Network Ocean Faucet",
           "desc": "The faucet to acquire your first Ocean test network tokens.",
           "url": "https://faucet.gaiaxtestnet.oceanprotocol.com/send?address=0x01e66950353400E93AEe7F041C0303103E2ef5Ab"
-        }
+        }, 
```

Comment:
The resource array from the previous category as well as the category then need to be closed here.
replace the `},` with
```
    }
  ]
},
```